modules/tracking/samples/lasot_benchmark.py

```diff
@@ -0,0 +1,193 @@
+import numpy as np
+import cv2 as cv
+import argparse
+import os
+import glob
+
+# Method used for evaluate corners coords 
+# Not used yet - need to re-write calculations with it
+def calc_coords(bb):
+    xmin = bb[0]
+    xmax = bb[0] + bb[2] - 1.0
+    ymin = bb[1]
+    ymax = bb[1] + bb[3] - 1.0
+    cx = bb[0] + (bb[2] + 1.0) / 2
+    cy = bb[1] + (bb[3] + 1.0) / 2
+    coords = {'xmin': xmin, 'xmax': xmax, 'ymin': ymin, 'ymax': ymax,
+                   'cx': cx, 'cy': cy}
+    return coords
```

Comment:
I think we can simplify this:
```python
return xmin, xmax, ymin, ymax, cx, cy
```
And below you can get:
```python
new_xmin, new_xmax, new_ymin, new_ymax, new_cx, new_cy = calc_coords(new_bb)
```

Reply:
Looks like centers are extra. Can calculate them in runtime?

Reply:
For now, centers are calculated only once for every bounding box, so if we remove their calculation in method, we will be forced to calculate them twice:
1) for precision;
2) for normalized precision.